StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/EmbeddedPaymentElementTest.swift

```diff
@@ -657,7 +705,118 @@ class EmbeddedPaymentElementTest: XCTestCase {
         // ...the delegate should be called after the form is submitted
         XCTAssertTrue(delegateDidUpdatePaymentOptionCalled, "Delegate should be updated after card form is completed")
         XCTAssertEqual(sut.paymentOption?.label, "•••• 4242")
+
+        await fulfillment(of: [rowSelectionBehaviorExpectation])
+    }
+
+    func testCreateFails_whenImmediateActionWithConfirmAndApplePay() async throws {
+        // Given a config that has rowSelectionBehavior = immediateAction, formSheetAction = .confirm, and Apple Pay
+        var config = configuration
```

Comment:
i'm confused here, because the computed variable `configuration` is:
```
    lazy var configuration: EmbeddedPaymentElement.Configuration = {
        var config = EmbeddedPaymentElement.Configuration._testValue_MostPermissive(isApplePayEnabled: false)
        config.apiClient = STPAPIClient(publishableKey: STPTestingDefaultPublishableKey)
        return config
    }()
```

but when i run it, apple pay configuration is set

Reply:
ah, `config` starts as a copy of `self.configuration`, but below on 719 is updated to add apple pay